pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -192,7 +192,11 @@ class TestCasting(BaseDecimal, base.BaseCastingTests):
 
 
 class TestGroupby(BaseDecimal, base.BaseGroupbyTests):
-    pass
+
+    @pytest.mark.xfail(
+        reason="needs to correctly define __eq__ to handle nans, xref #27081.")
+    def test_groupby_apply_identity(self, data_for_grouping):
+        super().test_groupby_apply_idendeity(data_for_grouping)
```

Comment:
Typo on identity if you care to fix on next push

Reply:
hmm let me push again